### PR TITLE
Add Fish

### PR DESCRIPTION
**Pull request by brj**
Add fish.

* https://github.com/fisherman/awesome-fish

Would love to have the awesome fish list here, but feel free to decline if this is out of the scope of awesome.

Cheers.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Yes, happy to have it, but I think the lists needs a little bit more work.

- Can you put all the badges in the same place (including the awesome badge)? Would look best on the right side of the main heading, but could also be below, but having it above looks messy.
- `Awesome fish-shell` => `Awesome fish shell`
- Add a contributing.md file. Example: https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md And link to it from the `Contributing` section.
- The repo description is not in sync with the readme one.
- You only list tools. Would be useful with some articles, learning material, videos, books, etc, about `fish`.
- Add some community resources. Example: https://github.com/sindresorhus/awesome-electron#community
- Not clear what the difference between `Classic` and `Powerline` is.
- Include the fish logo in the readme. Example: https://github.com/sindresorhus/awesome-electron
- Most of the `for fish-shell.` in the descriptions are moot.
- Drop the `build passing` badge. Not required, but it doesn't have much purpose. I realize you use Travis to check for valid PRs, but the badge doesn't have much purpose and is mostly just distracting.

**Comment by bucaran** [comment]
> Drop the build passing badge. Not required, but it doesn't have much purpose. I realize you use Travis to check for valid PRs, but the badge doesn't have much purpose and is mostly just distracting.

What about adding the build and slack room badge at the bottom?

**Comment by bucaran** [comment]
@sindresorhus 90% coverage of your requirement list.

> Include the fish logo in the readme.

There is none, or it's not well defined. The existing "unofficial" logo is too ugly.

> You only list tools. Would be useful with some articles, learning material, videos, books, etc, about fish.

Okay, I'll create a section for that and start adding content as I find it.

**Comment by sindresorhus** [comment]
> There is none, or it's not well defined. The existing "unofficial" logo is too ugly.

Doesn't look too bad: http://ridiculousfish.com/shell/images/fishfish_logo.png Maybe you could do something with it if you don't like it. Totally optional though. Just that some illustrations really help make an awesome list truly awesome. Example of an awesome header illustration: https://github.com/websemantics/awesome-synthetic-biology

**Comment by sindresorhus** [comment]
> Okay, I'll create a section for that and start adding content as I find it.

Maybe also some popular dotfiles repos using fish.

**Comment by bucaran** [comment]
> Maybe also some popular dotfiles repos using fish.

I'll look (again), but probably most of the things I have seen is people using a mix of bash/zsh with some fish nuggets, but not using fish as their main shell (which would be desirable in this case). 


> Doesn't look too bad: http://ridiculousfish.com/shell/images/fishfish_logo.png 

Haha. Anyway, I think something like awesome-synthetic-biology would be really special. I'll try to come up with a jumbo-like screen artwork.

**Comment by sindresorhus** [comment]
Btw, here's the command to print the logo: https://github.com/fish-shell/fish-shell/issues/114#issuecomment-24101750 So you can get it in any size ;)

**Comment by bucaran** [comment]
@sindresorhus Thanks for the link. I'll try to come up with an original cover art.

File: readme.md [hunk 1]
```diff
@@ -276,6 +276,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Dev Env](https://github.com/jondot/awesome-devenv)
 - [Dotfiles](https://github.com/webpro/awesome-dotfiles)
 - [Shell](https://github.com/alebcay/awesome-shell)
+- [Fish](https://github.com/fisherman/awesome-fish)
```
Comment (davisonio) [inline readme.md]: Could you update the link to the new repository location: https://github.com/brj/awesome-fish and also indent the bullet point so it is forms under the Shell list.

**Comment by sindresorhus** [comment]
ping :)

File: readme.md (hunk 1 above)
Comment (brj) [inline readme.md]: Fixed by moving the repo back to the org.

**Comment by brj** [comment]
@sindresorhus I didn't get around completing the artwork, because I was not happy with the result. 

[x] Can you put all the badg...
[x] Awesome fish-shell => Awesome fish shell
[x] Add a contributing.md f...
[x] The repo description is not in sync with the readme one.
[ ] You only list tools. Would be useful with some articles, ...
[x] Add some community resource...
[x] Not clear what the difference...
[ ] Include the fish logo in the.......
[x] Most of the for fish-shell...
[x] Drop the build passing badge. Not required, but it doesn't have much purpose. I realize you use Travis to check for valid PR...

File: readme.md (hunk 1 above)
Comment (brj) [inline readme.md]: @davisonio Fixed by moving the repo back to the org.

**Comment by sindresorhus** [comment]
Landed. Didn't want to wait anymore.

**Pull request merged**